capstone-frontend/yarn.lock

```diff
@@ -10494,9 +10494,9 @@ websocket-driver@>=0.5.1:
     websocket-extensions ">=0.1.1"
 
 websocket-extensions@>=0.1.1:
-  version "0.1.3"
-  resolved "https://registry.yarnpkg.com/websocket-extensions/-/websocket-extensions-0.1.3.tgz#5d2ff22977003ec687a4b87073dfbbac146ccf29"
-  integrity sha512-nqHUnMXmBzT0w570r2JpJxfiSD1IzoI+HGVdd3aZ0yNi3ngvQ4jv1dtHt5VGxfI2yj5yqImPhOK4vmIh2xMbGg==
+  version "0.1.4"
+  resolved "https://registry.yarnpkg.com/websocket-extensions/-/websocket-extensions-0.1.4.tgz#7f8473bc839dfd87608adb95d7eb075211578a42"
+  integrity sha512-OqedPIGOfsDlo31UNwYbCFMSaO9m9G/0faIHj5/dZFDMFqPTcx6UwqyOy3COEaEOg/9VsGIpdqn62W5KhoKSpg==
```

Comment:
I notice you have both a yarn.lock file and a package-lock.json file. You probably only need one of these. If you're using yarn, go with that. Otherwise, this can be deleted. Make sure everyone is on the same page about what tools you're using to build the project.

Reply:
Does the yarn.lock get created when you do a ```yarn build```? We are probably going to just use ```npm run build```.

Reply:
Ok, I delete yarn.lock from master